docker/openemr/flex-edge/utilities/openemr-cmd/openemr-cmd

```diff
@@ -0,0 +1,302 @@
+set_utf8_and_collation_to_utf8_general_ci(){
+	$DOCKER_EXEC_CMD $CONTAINER_ID $SHELL_TYPE '/root/devtools change-encoding-collation utf8 utf8_general_ci'
+} 
+
+USAGE_EXIT_CODE=13
+FINAL_EXIT_CODE=0
+
+# Confirm the docker install or not.
+DOCKER_CODE=16
+which docker &>/dev/null
+[ $? -ne 0 ] && echo "Please check docker install or not." && exit $DOCKER_CODE
+
+# Script usage.
+if [ $# -eq 0 ]
+then
+    echo "Usage: `basename $0` COMMAND [ARGS]"
+    echo 'Commands:'
+    echo "  build-themes                 Make changes to any files on your local file system"               
+	echo "  php-log                      To check PHP error logs"
+	echo "  psr12-report                 To create a report of PSR12 code styling issues"
+	echo "  psr12-fix                    To fix PSR12 code styling issues"
+	echo "  lint-themes-report           To create a report of theme styling issues"
+	echo "  lint-themes-fix              To fix theme styling issues" 
+	echo "  php-parserror                To check PHP parsing errors" 
+	echo "  unit-test                    To run unit testing"
+	echo "  api-test                     To run api testing"
+	echo "  e2e-test                     To run e2e testing"
+	echo "  services-test                To run services testing"
+	echo "  fixtures-test                To run fixtures testing"
+	echo "  validators-test              To run validators testing"
+	echo "  controllers-test             To run controllers testing"
+	echo "  common-test                  To run common testing"
+	echo "  clean-sweep                  To run the entire dev tool suite"
+	echo "  clean-sweep-tests            To run only all the automated tests" 
+	echo "  dev-reset                    To reset OpenEMR only"
+	echo "  dev-reset-install            To reset and reinstall OpenEMR"
+	echo "  dev-reset-install-demodata   To reset and reinstall OpenEMR with demo data"
+	echo "  backup-snapshot              Create a backup snapshot"
+	echo "  restore-snapshot             Restore from a snapshot"
+	echo "  list-snapshots               To list the snapshots"
+	echo "  list-capsules                List the capsules"
+	echo "  cp-cap-docker2dir            Copy the capsule from the docker to your current directory" 
+	echo "  cp-cap-dir2docker            Copy the capsule from current directory into the docker" 
+	echo "  ensure-version               Ensure run upgrade to ensure will work with current version OpenEMR"
+	echo "  enable-multisite             Turn on support for multisite"
+	echo "  disable-multisite            Turn off support for multisite"
+	echo "  utf8mb4-general-ci			 Set character set to utf8mb4 and collation to utf8mb4_general_ci"
+	echo "  utf8mb4-unicode-ci           Set character set to utf8mb4 and collation to utf8mb4_unicode_ci"
+	echo "  utf8mb4-vietnamese-ci		 Set character set to utf8mb4 and collation to utf8mb4_vietnamese_ci"
+	echo "  utf8-general-ci              Set character set to utf8 and collation to utf8_general_ci"
```

Comment:
Reduce the above 4 commands into 1 and then use 2 parameters for the encoding and collation. this command would look something like:
```
openemr-cmd encoding-collation utf8mb4 utf8mb4-general-ci
```
(this way, the literally hundred or so combinations of encoding and collations are supported with one command)

Reply:
btw, if wish to keep shortcut commands like you have for the 3 main calls (would drop the vietnamese one), then that would also make a lot of sense to keep. But should also have a generic command to easily change to the other ones.